frontend/playwright.config.ts

```diff
@@ -54,7 +54,7 @@ const config: PlaywrightTestConfig = {
 
     /* See https://playwright.dev/docs/trace-viewer */
     /* not 'on-first-retry', because then you don't have good traces to reference when debugging */
-    trace: 'on',
+    trace: 'retain-on-failure',
```

Comment:
I really prefer `on` as my comment on the line above (somewhat poorly) explains.
I've been in too many situations where I wanted to see what a successful trace looked like as part of debugging an unsuccessful execution. So, I like having successful traces available.

Reply:
I'll try setting this back to `on`, but it might result in no traces at all because the `7z` process runs out of memory and gets killed again. The `trace.zip` files are quite large. However, my setting 7z to use no compression may have fixed that problem; we'll see.

Reply:
Done in commit 014af79e7b9fec6f6cab37cdf74e289ffa18b205, and the 7z process did not get OOM killed. Looks like this is working.